frontend/test/metabase/scenarios/visualizations/drillthroughs/chart_drill.cy.spec.js

```diff
@@ -51,13 +51,13 @@ describe("scenarios > visualizations > drillthroughs > chart drill", () => {
       cy.wait(100); // wait longer to avoid grabbing the svg before a chart redraw
 
       // drag across to filter
-      cy.get(".dc-chart svg")
+      cy.get(".Visualization")
         .trigger("mousedown", 100, 200)
         .trigger("mousemove", 200, 200)
         .trigger("mouseup", 200, 200);
 
       // new filter applied
-      cy.contains("Created At between May, 2016 July, 2016");
+      cy.contains("Created At between May, 2016 September, 2016");
```

Comment:
![image](https://user-images.githubusercontent.com/784417/113774008-57895b80-96ec-11eb-93a2-d0df739c8abf.png)
why change the month? It's failing locally for me

Reply:
Because "September" is what I've got locally as a result.

This test was failing because `mouseover` was being triggered on `svg` (which was apparently rerendering - causing the detached error). So, I decided to use the container `.Visualization` as a reference point. That should've changed the coordinates (range) a bit, and it was logical to me that month also slightly changed.

Makes sense?

Reply:
yeah, makes sense, just a little curious I couldn't repro it locally. Computers!

Well, CI is the real source of truth here. I'll give you a :heavy_check_mark: contingent on CI passing; the code looks reasonable